src/sqlfluff/dialects/dialect_tsql.py

```diff
@@ -2345,24 +2349,27 @@ class TransactionStatementSegment(BaseSegment):
 
     type = "transaction_statement"
     match_grammar = OneOf(
-        # BEGIN | SAVE TRANSACTION
-        # COMMIT [ TRANSACTION | WORK ]
-        # ROLLBACK [ TRANSACTION | WORK ]
+        # [ BEGIN | SAVE ] [ TRANSACTION | TRAN ]
+        # COMMIT [ TRANSACTION | TRAN | WORK ]
+        # ROLLBACK [ TRANSACTION | TRAN | WORK ]
```

Comment:
```suggestion
        # [ BEGIN | SAVE ] [ TRANSACTION | TRANS ]
        # COMMIT [ TRANSACTION | TRANS | WORK ]
        # ROLLBACK [ TRANSACTION | TRANS | WORK ]
```

Shouldn't these be `TRANS`?

Reply:
Also maybe include the `SAVE` version of these for consistency.

Reply:
Save is already there, next to `BEGIN`  🙂

Reply:
Oh lol sorry! 😄